### DISCUSSION TITLE
Folder icon for STACK folder

### DISCUSSION THREAD

**Discussion by joostiphone** [elrumo/macOS_Big_Sur_icons_replacements#1369] [Icon request]
Combining the [default folder icon](https://media.macosicons.com/parse/files/macOSicons/7ede37438a53b4ed6c833f7b8d8b097d_1619092574091.icns) with this one:

![stack-250px-600x600](https://user-images.githubusercontent.com/16277235/159431167-4251514c-7056-4fdb-91d8-edf6f0a674c7.png)

Result example: https://media.macosicons.com/parse/files/macOSicons/e931601ea8878996cb530d6ebc65411f_1621943560909.icns

Thanks

**Comment by di0xazin-e** [comment]
https://imgur.com/a/Q5uQPTE

Here you go man! Hope you enjoy this and also it will be on my page soon.
https://macosicons.com/#/u/alivat0r

**Reply by joostiphone** [reply]
Thanks! Looks great, awesome.
